### PR TITLE
SC Data release 10 2020-05

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds the release notes for SC Data release 2020-05, highlighting covid datasets.

### CODE REVIEW

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- No new species, the aim of this release was to focus on COVID-19 related datasets.
+
+#### **?** experiments relevant to the research of the current **COVID-19** outbreak caused by the **coronavirus SARS-Cov-2**.
+- [A cellular census of human lungs identifies novel cell states in health and asthma](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-130148/results)
+- [Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-9/results)
+- [Single-cell reconstruction of the early maternal-fetal interface in humans - Droplet](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6701/results)
+- [Single-cell reconstruction of the early maternal-fetal interface in humans - SmartSeq2](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6678/results)
+- [Spatiotemporal immune zonation of the human kidney](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-10/results)
+- [Individual Variation of the SARS-CoV2 Receptor ACE2 Gene Expression and Regulation](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-83139/results)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md]: Experiment title should be: Single cell RNA-seq of human pancreatic endocrine cells from juvenile, adult control and type 2 diabetic donors

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#### Data statistics
+- Ensembl **99** / Ensembl Genomes **46** / WormBase ParaSite **14** gene annotations.   
+- This release contains **164** single cell RNA-Seq studies, consisting of **4,646,599** cells, of which **3,342,629** passed our QC from **14** different species.
+- No new species, the aim of this release was to focus on COVID-19 related datasets.
+
+#### **?** experiments relevant to the research of the current **COVID-19** outbreak caused by the **coronavirus SARS-Cov-2**.
+- [A cellular census of human lungs identifies novel cell states in health and asthma](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-130148/results)
+- [Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-9/results)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md]: Here you've used the paper titles, not the experiment titles. I think we should change that to the experiment titles. 
E-GEOD-130148: "Lung Cell Atlas: Single cell RNA sequencing analysis of fresh resected human lung tissue - Drop-seq dataset"
E-HCAD-9: "Dissecting the human liver cellular landscape by single cell RNA-seq reveals novel intrahepatic monocyte/macrophage populations"

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- [Single-cell transcriptomics of the human retinal pigment epithelium and choroid in health and macular degeneration](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-135922/results/)
+- [Human fetal liver, skin and kidney single cell transcriptome data](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7407/results/)
+- [scRNA-seq of human fetal lung primary tissues and cell cultures derived from fetal bud tip progenitors under dual SMAD treatment](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8221/results/)
+- [Ischaemic sensitivity of human tissue by single cell RNA seq](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-1/results/)
+
+
+#### New experiments
+- [Single-cell analysis of Crohn's disease lesions identifies a pathogenic cellular module associated with resistance to anti-TNF therapy](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-46)
+- [Single-cell transcriptome analysis reveals differential nutrient absorption functions in human intestine](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970)
+- [The Dynamic Transcriptional Cell Atlas of Testis Development during Human Puberty](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-134144)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md:30]: This one was also in the Covid-19 list.

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+- [Single Cell RNA-sequencing of murine tail interfollicular epidermis basal cells at postnatal days 7](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-146122)
+- [Defining the Activated Fibroblast Population in Lung Fibrosis Using Single Cell Sequencing](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-14)
+- [Transcriptomic Profiling of the Developing Cardiac Conduction System at Single-Cell Resolution](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-16)
+- [A Cellular Atlas of Pitx2-Dependent Cardiac Development](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-17)
+- [An integrated approach to profile lung tumor endothelial cell heterogeneity across species and models and to identify angiogenic candidates](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6308)
+- [Single cell RNA-seq of a human ex vivo ovarian model with matched stromal cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7249)
+- [scRNA-seq analysis of lung CD64-expressing mononuclear cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7678)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md]: Title is truncated. Should be: "scRNA-seq analysis of lung CD64-expressing mononuclear cells, patrolling and classical monocytes from steady-state C57BL/6J mice"

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- [scRNA-seq of human fetal lung primary tissues and cell cultures derived from fetal bud tip progenitors under dual SMAD treatment](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8221)
+- [Single cell RNA-seq from 10X for a living biobank of ovarian cancer ex vivo models reveals profound mitotic heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8559)
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md:39]: This one is related to E-MTAB-7249 and they could be next to each other :)

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+#### Data statistics
+- Ensembl **99** / Ensembl Genomes **46** / WormBase ParaSite **14** gene annotations.   
+- This release contains **164** single cell RNA-Seq studies, consisting of **4,646,599** cells, of which **3,342,629** passed our QC from **14** different species.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md:3]: I see 165 studies now

File: release-notes/sc/_posts/2020-05-14-10.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- [An integrated approach to profile lung tumor endothelial cell heterogeneity across species and models and to identify angiogenic candidates](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6308)
+- [Single cell RNA-seq of a human ex vivo ovarian model with matched stromal cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7249)
+- [scRNA-seq analysis of lung CD64-expressing mononuclear cells](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7678)
+- [scRNA-seq of human fetal lung primary tissues and cell cultures derived from fetal bud tip progenitors under dual SMAD treatment](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8221)
+- [Single cell RNA-seq from 10X for a living biobank of ovarian cancer ex vivo models reveals profound mitotic heterogeneity](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-8559)
+
+### Experiments temporarily removed
+
+Due to changes in the GDPR state of the experiment, the following entry was removed:
+- HDCA project: single cell transcriptomics (E-HCAD-7). We are in the process of requesting access to its set Data Access Committee.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-05-14-10.md]: "through the access data committee"? What about "We are in the process of requesting data access to reinstate the Expression Atlas study."?

**Review by anjaf — CHANGES_REQUESTED** [review]
Generally fine. A few small things to update.

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 6 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md:3]: (waving hand) now you see 164 ;-)

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 7 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]: have you tried the new github suggestions?

```suggestion
- HDCA project: single cell transcriptomics (E-HCAD-7). We are in the process of requesting data access to reinstate the Expression Atlas study.
```

**Comment by pcm32** [comment]
Thanks @anjaf !

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 1 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]: Sure, I can change this on, but this is just retrieved from the `.experimentDescription` of the `https://wwwdev.ebi.ac.uk/gxa/sc/json/experiments` endpoint. So I guess that we either need an additional field returned from that endpoint @alfonsomunozpomer with the correct field that we should be using (title?) or maybe who curated this experiment used that field (description) in an usual manner @anjaf ?

Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- [Single cell RNA-seq of human pancreatic endocrine cells from juvenile, adult control and type 2 diabetic donors](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-83139/results)
```

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 2 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]: Will do, again same comment as in the other one, lets make sure that curators use the field consistently here I guess or let us know if we need a new field for this.

Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- [Lung Cell Atlas: Single cell RNA sequencing analysis of fresh resected human lung tissue - Drop-seq dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-130148/results)
- [Dissecting the human liver cellular landscape by single cell RNA-seq reveals novel intrahepatic monocyte/macrophage populations](https://www.ebi.ac.uk/gxa/sc/experiments/E-HCAD-9/results)
```

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 3 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md:30]: Only E-GEOD-134144 correct? or the 3 of them?

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 4 above)
Comment (pcm32) [reply release-notes/sc/_posts/2020-05-14-10.md]:
```suggestion
- [scRNA-seq analysis of lung CD64-expressing mononuclear cells, patrolling and classical monocytes from steady-state C57BL/6J mice](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-7678)
```

File: release-notes/sc/_posts/2020-05-14-10.md (hunk 3 above)
Comment (anjaf) [reply release-notes/sc/_posts/2020-05-14-10.md:30]: Yes, just E-GEOD-134144

**Pull request merged**